src/acceptance-test/java/org/zalando/nakadi/webservice/utils/TestStreamingClient.java

```diff
@@ -28,6 +29,7 @@
     private final List<StreamBatch> batches;
     private InputStream inputStream;
     private String sessionId;
+    private Optional<String> token;
```

Comment:
No, it's not possible to make it final. In the second constructor I reassign its value.